package.json

```diff
@@ -15,7 +15,7 @@
     "test:coverage": "jest --coverage",
     "test:smoke": "NODE_TLS_REJECT_UNAUTHORIZED=0 jest -c jest.smoke.config.js",
     "test:crossbrowser": "npm-run-all -scl test:crossbrowser:e2e test:crossbrowser:report",
-    "test:crossbrowser:e2e": "NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs --config src/test/cross-browser/saucelabs.conf.js run-multiple --all",
+    "test:crossbrowser:e2e": "NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs --config src/test/cross-browser/saucelabs.conf.js run-multiple --all --grep \"(.*@CrossBrowser)\"",
```

Comment:
Think you can simplify the grep part to below. If you are checking for the tag so no need for the regex
--grep @CrossBrowser

Reply:
okay